site/content/en/docs/manual/advanced/cli.md

```diff
@@ -46,13 +57,151 @@ optional arguments:
   --debug               show debug output
 ```
 
-**Examples**
+You can get help for each positional argument, e.g. `ls`:
+
+```bash
+./cli.py ls -h
+```
+```
+usage: cli.py ls [-h] [--json]
+
+List all CVAT tasks in simple or JSON format.
+
+optional arguments:
+  -h, --help  show this help message and exit
+  --json      output JSON data
+```
+
+## Examples
+
+- Create a task named "new task", labels from the file "labels.json" and local images "file1.jpg" and "file2.jpg":
+  ```bash
+  cli.py create "new task" --labels labels.json local file1.jpg file2.jpg
+  ```
+- Create a task named "task 1", labels from the project with id 1 and with a remote video file,
+  the task will be created as user "user-1":
+  ```bash
+  cli.py --auth user-1:password create "task 1" --project_id 1 \
+  remote https://github.com/opencv/opencv/blob/master/samples/data/vtest.avi?raw=true
+  ```
+- Delete tasks with id "100", "101", "102":
+  ```bash
+  cli.py delete 100 101 102
+  ```
+- List all tasks:
+  ```bash
+  cli.py ls
+  ```
+- Dump annotation task with id 103, in the format "CVAT for images 1.1" and save to the file "output.xml":
+  ```bash
+  cli.py dump --format "CVAT for images 1.1" 103 output.xml
+  ```
+
+## Arguments
```

Comment:
@TOsmanov , I don't think we need repeat here the help message from `cli.py`. It is useless. Instead I would recommend to extend the section with examples and split them by functionality. It is really better to explain some parameters by examples.

### Create

- Create a task named "new task", labels from the file "labels.json" and local images "file1.jpg" and "file2.jpg":
  ```bash
  cli.py create "new task" --labels labels.json local file1.jpg file2.jpg
  ```
- Create a task named "task 1", labels from the project with id 1 and with a remote video file,
  the task will be created as user "user-1":
  ```bash
  cli.py --auth user-1:password create "task 1" --project_id 1 \
  remote https://github.com/opencv/opencv/blob/master/samples/data/vtest.avi?raw=true
  ```

### Delete

- Delete tasks with id "100", "101", "102":
  ```bash
  cli.py delete 100 101 102
  ```